test/extended/builds/run_policy.go

```diff
@@ -16,7 +16,8 @@ import (
 	exutil "github.com/openshift/origin/test/extended/util"
 )
 
-var _ = g.Describe("[builds][Slow] using build configuration runPolicy", func() {
+// this test is very latency sensitive so run it by itself (serially).
+var _ = g.Describe("[builds][Slow][Serial] using build configuration runPolicy", func() {
```

Comment:
Since you're already making this change, might as well get rid of the assignment to `var _`. I don't see the need for that.

Reply:
hm. it's what we do in all our extended tests(not just the builds ones), I assumed it served some initialization purpose/magic.

Reply:
"The var _ = ... trick allows us to evaluate the Describe at the top level without having to wrap it in a func init() {}"

(from https://onsi.github.io/ginkgo/)

Reply:
ahh, true you don't need an init. Thx.